clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -786,4 +786,7 @@ def warn_android_unversioned_fallback : Warning<
   " directories will not be used in Clang 19. Provide a versioned directory"
   " for the target version or lower instead.">,
   InGroup<DiagGroup<"android-unversioned-fallback">>;
+
+def err_android_version_invalid : Error<
+  "Version %0 in triple %1 is invalid.">;
```

Comment:
No capitalization and drop the full stop to be consistent with other messages/convention.

Reply:
* Elsewhere we say "target triple"
* Repeating the version seems a bit ugly?

Reply:
Perhaps `version '%0' in target triple '%1' is invalid`